packages/storybook/package.json

```diff
@@ -43,6 +43,7 @@
     "babel-runtime": "^6.26.0",
     "concurrently": "^3.5.1",
     "enzyme": "^2.9.1",
+    "faker": "^4.1.0",
```

Comment:
Let's move this into the `devDependencies` for the `@hig/table` package. Although it's being used for for Storybook, we like to keep the dev dependencies for each package separate.